airbyte-integrations/connectors/source-faker/source_faker/spec.json

```diff
@@ -35,8 +35,16 @@
         "description": "How many fake records will be in each page (stream slice), before a state message is emitted?",
         "type": "integer",
         "minimum": 1,
-        "default": 100,
+        "default": 1000,
         "order": 3
+      },
+      "threads": {
+        "title": "Worker Threads",
+        "description": "How many parallel worker threads should be used to generate records?",
```

Comment:
if we must keep this optino, could you add some guidance on how one should choose this number? (or at least tell me if I don't know what this is doing, just ignore it)